patterns/disposer.md

```diff
@@ -0,0 +1,84 @@
+# The Promise Disposer Pattern
+
+Often we have to perform resource management in code. For example - release a DB connection:
+
+
+```js
+var users = getDb().then(conn => 
+    conn.query("SELECT name FROM users").then(users => {
```

Comment:
I'll convert it to a multi-line lambda for clarify - that's how the node docs behave so I might as well align - thanks.